src/views/Tree.vue

```diff
@@ -0,0 +1,159 @@
+<template>
```

Comment:
Just a really simple tree view for now, I needed a second view to test query merging, view registering / unregistering etc. #78 can rip this template out.

Reply:
Excellent! #78 just got much easy to implement. Thanks!